test/core/security/ssl_server_fuzzer.cc

```diff
@@ -91,7 +91,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
     struct handshake_state state;
     state.done_callback_called = false;
     grpc_handshake_manager* handshake_mgr = grpc_handshake_manager_create();
-    grpc_server_security_connector_add_handshakers(sc, handshake_mgr);
+    grpc_server_security_connector_add_handshakers(sc, nullptr, handshake_mgr);
     grpc_handshake_manager_do_handshake(
```

Comment:
We should also remove the `interested_parties` argument from the `grpc_handshake_manager_do_handshake()` method, since it's no longer needed there.

Reply:
Yes, I will have a proceeding PR that does the job of removing `interested parties` from `grpc_handshake_manager_do_handshake()`.